sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosItemRequestOptions.java

```diff
@@ -175,7 +171,7 @@ public CosmosItemRequestOptions setPartitionKey(PartitionKey partitionKey) {
      *
      * @return the partition key
      */
-    public PartitionKey getPartitionKey() {
+    PartitionKey getPartitionKey() {
```

Comment:
the public setter for partition key needs to be removed too because the partition key is a mandatory standalone param on reading/deleting item and doesn't have to be in the cosmosItemRequestOption anymore

Reply:
Removed